web/src/main/java/org/openmrs/web/filter/initialization/DatabaseDetective.java

```diff
@@ -40,7 +42,7 @@ public boolean isDatabaseEmpty(Properties props) {
 		Connection connection = null;
 		
 		try {
-			DatabaseUtil.loadDatabaseDriver(props.getProperty(CONNECTION_URL), null);
+			DatabaseUtil.loadDatabaseDriver(props.getProperty(CONNECTION_URL), props.getProperty(CONNECTION_DRIVER_CLASS,null));
```

Comment:
Our convention is to add space after the comma

Reply:
thanks @dkayiwa